File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.py
def solution(numLog):
    answer = ''
    tmp = 0
    
    for i in range(1,len(numLog)):
        tmp = int(numLog[i]) - int(numLog[i-1])
        if tmp == 1: answer += 'w'
        elif tmp == -1: answer +='s'
        elif tmp == 10: answer +='d'
        elif tmp == -10: answer += 'a'
        
    return answer

File: 프로그래머스/0/181925. 수 조작하기 2/수 조작하기 2.java
class Solution {
    public String solution(int[] numLog) {
        String answer = "";
        int tmp = 0;
        
        for(int i=1;i<numLog.length;i++){
            tmp = numLog[i] - numLog[i-1];
            if(tmp == 1) answer += "w";
            else if(tmp == -1) answer += "s";
            else if(tmp == 10) answer += "d";
            else if(tmp == -10) answer += "a";
        }
        
        return answer;
    }
}